libethereum/Executive.cpp

```diff
@@ -255,13 +256,16 @@ bool Executive::call(Address _receiveAddress, Address _senderAddress, u256 _valu
 
 bool Executive::call(CallParameters const& _p, u256 const& _gasPrice, Address const& _origin)
 {
-	// Always remember the sender, needed for revert.
-	m_revertLog.caller = _p.senderAddress;
-
 	// If external transaction.
 	if (m_t)
+	{
+		// FIXME: some unrevertable changes occupy change log.
```

Comment:
I checked what's there exactly -- it is balance change that paid for the transaction.